src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -226,47 +226,39 @@ and a guide for using macros can be found at
-| DBS              | Electrode channel                                    |
-| VEOG             | Vertical EOG (electrooculogram)                      |
-| HEOG             | Horizontal EOG                                       |
-| EOG              | Generic EOG channel                                  |
-| ECG              | ElectroCardioGram (heart)                            |
-| EMG              | ElectroMyoGram (muscle)                              |
-| TRIG             | System Triggers                                      |
-| AUDIO            | Audio signal                                         |
-| PD               | Photodiode                                           |
-| EYEGAZE          | Eye Tracker gaze                                     |
-| PUPIL            | Eye Tracker pupil diameter                           |
-| MISC             | Miscellaneous                                        |
-| SYSCLOCK         | System time showing elapsed time since trial started |
-| ADC              | Analog to Digital input                              |
-| DAC              | Digital to Analog output                             |
-| HLU              | Measured position of head and head coils             |
-| FITERR           | Fit error signal from each head localization coil    |
-| OTHER            | Any other type of channel                            |
+| **Keyword**      | **Description**                                             |
+| ---------------- | ----------------------------------------------------------- |
+| MEGMAG           | MEG magnetometer                                            |
+| MEGGRADAXIAL     | MEG axial gradiometer                                       |
+| MEGGRADPLANAR    | MEG planargradiometer                                       |
+| MEGREFMAG        | MEG reference magnetometer                                  |
+| MEGREFGRADAXIAL  | MEG reference axial gradiometer                             |
+| MEGREFGRADPLANAR | MEG reference planar gradiometer                            |
+| MEGOTHER         | Any other type of MEG sensor                                |
+| EEG              | Electrode channel                                           |
+| ECOG             | Electrode channel                                           |
+| SEEG             | Electrode channel                                           |
+| DBS              | Electrode channel                                           |
+| VEOG             | Vertical EOG (electrooculogram)                             |
+| HEOG             | Horizontal EOG                                              |
+| EOG              | Generic EOG channel                                         |
+| ECG              | ElectroCardioGram (heart)                                   |
+| EMG              | ElectroMyoGram (muscle)                                     |
+| TRIG             | Analog (in volt) or digital (TTL bit level) trigger channel |
```

Comment:
The "TTL" convention applies to both the analog representation as the binary interpretation. Even in memory of a computer a bit is still represented as a voltage. 

I suggest to write "Analog (TTL in Volt) or digital (binary TTL) trigger channel".

Reply:
oh, and I would not write "bit level", as that suggests that every bit is to be represented in a separate channel. Some systems do that (like Neuromag, only for backward compatibility), but it is not so common. Most represent the bits jointly in a single numeric value.

Reply:
thanks! implemented in https://github.com/bids-standard/bids-specification/pull/1342/commits/eaa0dfb990b0f6bee5f55e82dbb356a14f69b603